include/AJAX_Requests.php

```diff
@@ -0,0 +1,31 @@
+<?php
+include('../include/initialize.php'); 
+
+//I'm planning to send all my AJAX requests to this file from now as I get more familiar with the concept. Is this ok?
+//How do you usually do this?
+
+if(isset($_GET['StudentIdToEdit'])){
+    $studentId = $_GET['StudentIdToEdit'];
+
+    echo json_encode(getOneStudent($studentId));
+} 
+
+if(isset($_POST['SavePrivateNotes'])){
+
+    $commments = json_decode($_POST['SavePrivateNotes']);
```

Comment:
You don't have to change this, but if you'd rather work with a PHP array instead of a generic PHP object, you can pass `true` to the second parameter of `json_decode` and it'll create an array instead. It's mostly a preference thing, but I tend to prefer using an array over an object personally.

Reply:
Great! I also preferer arrays.